scripts/setup-project.sh

```
@@ -0,0 +1,750 @@
+    # Create a temporary file for the value
+    local tmp_file=$(mktemp)
+    echo "$value" > "$tmp_file"
+    
+    # Try to set the environment variable in Vercel
+    if ! vercel env rm "$key" "$scope" -y > /dev/null 2>&1; then
+        echo -e "${YELLOW}Note: $key doesn't exist in Vercel yet${NC}"
+    fi
+    
+    # For public variables, set them separately for each environment
+    if [[ "$scope" == *","* ]]; then
+        local environments=(${scope//,/ })
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)